packages/dataviz/src/components/BoxPlot/BoxPlot.component.test.tsx

```diff
@@ -12,7 +12,7 @@ describe('Boxplot chart', () => {
 					q1: 8,
 					q2: 90,
 					median: 58,
-					mean: 59.79
+					mean: 59.79,
```

Comment:
Is it intended ?

Reply:
I applied prettier on the whole src directory, do you mean the prettier config is not correct?

Reply:
Yes, it is correct. The goal is to facilitate reading of the diff when an attribute is added.